.eslintrc.json

```diff
@@ -10,48 +10,53 @@
     "next",
     "prettier"
   ],
-  "plugins": ["@typescript-eslint", "prettier", "unicorn", "lit-a11y"],
```

Comment:
Try to remove the eslint changes from the PR if possible. If this still doesn't work please dm me and I'll try to get it fixed.